routes/index.js

```diff
@@ -1,5 +1,9 @@
 const router = require("express").Router();
 
 router.use("/api", require("./api"));
+// return for call to /api
+router.get("/api", (req, res) => {
```

Comment:
Why did you edit this route? This update will cut off entry to all the other routes and controllers, please attend to it.